src/fragments/start/getting-started/flutter/build.mdx

```diff
@@ -1,26 +1,23 @@
 ## What you'll build
 
-The stable release of Amplify Flutter currently supports iOS/Android as target platforms.
+Amplify Flutter supports all Flutter platforms as targets (iOS, Android, Web, Windows, macOS, and Linux).
 
 <Callout warning>
 
-Web and desktop support for the Amplify Flutter Auth, API, Analytics and Storage categories are now in developer preview!
-
-However, this getting started guide uses the stable release.
-
-See the documentation for [project setup](/lib/project-setup/create-application) and the [Auth category](/lib/auth/getting-started), the API category ([GraphQL](/lib/graphqlapi/getting-started) and [REST](/lib/restapi/getting-started)), the [Analytics category](/lib/analytics/getting-started), and the [Storage category](/lib/storage/getting-started) for more information about web and desktop support.
-
+The DataStore and Notifications categories only support iOS/Android currently. Please open an issue in our
+[GitHub](https://github.com/aws-amplify/amplify-flutter/) with your use case if you'd like to see support for 
```

Comment:
can we please remove the link to the Github and the recommendation to create the issue for now until we have specific ones to link to. Want to reduce duplication of tickets created.